jolly-jellyfish/src/page_maker/templates/registration/register.html

```diff
@@ -0,0 +1,17 @@
+{% extends "page_maker/base.html" %}
+{% block content %}
+   <div>
```

Comment:
```suggestion
{% block content %}
   <h2>Sign up</h2>
   <div>
```
As above, styling concerns.